jans-pycloudlib/tests/test_cli.py

```
@@ -64,6 +64,11 @@ def test_encoding_decode_file_no_salt(monkeypatch, gmanager):
         lambda: gmanager,
     )
 
+    monkeypatch.setattr(
+        "hvac.Client.is_authenticated",
+        lambda cls: True,
```

Ruff (0.15.9)

[warning] Unused lambda argument: `cls`

(ARG005)